README.md

```diff
@@ -35,6 +35,10 @@ Hub itself.
 Contributions to the process happen in this repository while the software powering
 the Hub is [Monocular](https://github.com/helm/monocular), a Helm project.
 
+## Happy helming in China
```

Comment:
Maybe:

If you are in China, there are some issues to use `https://hub.helm.sh` directly, e.g. images hosted on `gcr.io`, `quary.io`, and Charts hosted on `googleapis.com` etc. We recommend you use this mirror hub at https://developer.aliyun.com/hub which will automatically sync & replace unavailable image & repo URLs in every Chart.